### ISSUE TITLE
Cannot build by Docker on Mac OS

### ISSUE THREAD

**Issue by Nebulazhang** [kubernetes/kubernetes#75031]
**What happened**:
I pull master and `build/run.sh make` but failed.
I met an error `k8s.io/kubernetes/hack/lib/util.sh: line 654: DOCKER[@]: unbound variable`

https://github.com/kubernetes/kubernetes/blob/9c75270b2be4dcc39e31edb34b76e8f604aaf91a/hack/lib/util.sh#L651

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
Mac
- Kernel (e.g. `uname -a`):
- Install tools:
- Others:
```
 ~  docker version
Client: Docker Engine - Community
 Version:           18.09.1
 API version:       1.39
 Go version:        go1.10.6
 Git commit:        4c52b90
 Built:             Wed Jan  9 19:33:12 2019
 OS/Arch:           darwin/amd64
 Experimental:      false

Server: Docker Engine - Community
 Engine:
  Version:          18.09.1
  API version:      1.39 (minimum version 1.12)
  Go version:       go1.10.6
  Git commit:       4c52b90
  Built:            Wed Jan  9 19:41:49 2019
  OS/Arch:          linux/amd64
  Experimental:     false
```

**Comment by mrbobbytables** [comment]
This was accidentally introduced not too long ago in [#74257](https://github.com/kubernetes/kubernetes/pull/74257#discussion_r262554139) and should be resolved by #74970 .
Would you mind pulling down a more recent copy and trying again?
/assign
/sig testing

**Comment by mrbobbytables** [comment]
I think this is safe to close at this point. If you have problems. Please feel free to reopen.

/close
